Consistence/Sniffs/Exceptions/ExceptionNameEndsWithExceptionSniff.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Consistence\Sniffs\Exceptions;
+
+use PHP_CodeSniffer_File;
+use SlevomatCodingStandard\Helpers\ClassHelper;
+use StringHelper;
+
+class ExceptionNameEndsWithExceptionSniff implements \PHP_CodeSniffer_Sniff
```

Comment:
As we have discussed, I propose `ExceptionDeclarationSniff`, which will (in the future) check all the rules considering exception declaration as stated in the standard. This will ease the configuration of possible options regarding exceptions. And the rules will each have their own code, so it should not be a problem to pick the ones which do/don't suit you.